samples/core/console-apps/FibonacciBetterMsBuild/README.md

```diff
@@ -0,0 +1,25 @@
+Better Fibonacci Sample
+================
+
+This sample is part of the [step-by-step tutorial](https://docs.microsoft.com/dotnet/articles/core/preview3/tutorials/using-with-xplat-cli-msbuild)
+for creating .NET Core Console Applications. Please see that topic for detailed steps on the code
+for this sample.
+
+Key Features
+------------
+
+This sample demonstrates a multiple source file project, adding recursion and caching previous
+results to the initial Fibonacci sample.
+
+Build and Run
+-------------
+
+To build and run the sample, type the following three commands:
+
+`dotnet restore`
+`dotnet build`
+`dotnet run`
+
+`dotnet restore` installs all the dependencies for this sample into the current directory.
```

Comment:
Not true, better would be "restores the dependencies for this sample". They don't really go into the current directory and maybe at this point we don't care where they go?